source-map-support.js

```diff
@@ -84,7 +84,7 @@ function retrieveSourceMapURL(source) {
 
   // Get the URL of the source map
   fileData = retrieveFile(source);
-  var re = /\/\/[#@]\s*sourceMappingURL=([^'"]+)\s*$/mg;
+  var re = /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
```

Comment:
It is already better with their comment above the regexp but I don't understand it fully, especially the `{1}`…
I wonder if this regexp should not be in this own package like [ip-regex](https://github.com/sindresorhus/ip-regex).

Anyway, are you sure all tests pass, even those [in browsers](/evanw/node-source-map-support#tests)?